app/models/spree/shipping_method_decorator.rb

```diff
@@ -30,6 +30,7 @@
   }
 
   scope :by_name, -> { order('spree_shipping_methods.name ASC') }
+  scope :display_on_checkout, -> { where("display_on is null OR display_on = ''") }
```

Comment:
It's better to include the table name `spree_shipping_methods` in case we join other tables which have the same column (in the future). But this is much easier to read. I'm happy to leave this as is until future versions of Rails make it `or` easier.

Reply:
I thought about this actually and decided to leave it like this for simplicity... yeah, I am not sure...

Reply:
I think adding the table name explicitly is a good idea

Reply:
ok, I have added it :+1: